.github/workflows/ci.yml

```diff
@@ -49,15 +43,73 @@ jobs:
 
     - name: Run Memgraph
       run: |
-        docker run -d -p 7687:7687 -v mg_lib:/var/lib/memgraph -v mg_log:/var/log/memgraph -v mg_etc:/etc/memgraph memgraph
+        docker run -d -p 7687:7687 memgraph --telemetry-enabled=False
 
     - name: Run cargo-tarpaulin
-      uses: actions-rs/tarpaulin@v0.1.2
+      uses: actions-rs/tarpaulin@v0.1.3
       with:
-        args: '--out Html --ignore-tests'
-
+        args: '--out Html --ignore-tests --exclude-files mgclient/*'
     - name: Archive code coverage results
       uses: actions/upload-artifact@v1
       with:
         name: code-coverage-report
         path: tarpaulin-report.html
+
+  build_macos:
+    strategy:
+      matrix:
+        platform: [macos-latest]
+        target: [x86_64-apple-darwin]
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+    - name: Install Rustup
+      run: |
+        curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup-init.sh
+        sh rustup-init.sh -y --default-toolchain none
+        rustup target add ${{ matrix.target }}
+
+    - uses: actions/checkout@v2
+      with:
+          submodules: true
+
+    - name: Build the client
+      run: cargo build --release
+
+  build_windows:
+    strategy:
+      matrix:
+        platform: [windows-2019]
+        target: [x86_64-pc-windows-gnu]
+        arch:
+          - { mingw: "64", msys: x86_64 }
```

Comment:
I actually took that from mgclient, so that was a mistake from way back, but yea it's not consistent, I even think we don't quote most of the stuff, do we put `"` or we just remove `"` in the 64 case?

Reply:
I hope this makes sense now :smile: